backend/src/config/general.rs

```diff
@@ -53,7 +53,7 @@ pub(crate) struct GeneralConfig {
     ///
     /// ```
     /// footer_links = [
-    ///     { label = { en = "Example" }, link = "https://example.com" },
+    ///     { label = { en = "Example" }, link = { en = "https://example.com/en" } },
```

Comment:
Can you add another link in this example that does not use translated link? And please add one sentence above this example saying "The link can either be the same for all languages or specified per language." or something like that.